bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/contract/form/mapper/FormDetailsMapper.java

```diff
@@ -0,0 +1,58 @@
+package org.bahmni.module.bahmnicore.contract.form.mapper;
```

Comment:
Same comment about packages as above. This can be just under "org.bahmni.module.bahmnicore.mapper" package.